examples/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.commercetools" % "sbt-scraml" % "0.8.5+5-e3f907e8+20210915-1002-SNAPSHOT")
```

Comment:
this will need to be changed to the released after merge. you can use `publishLocal` and replace the version for the review.